### ISSUE TITLE
Sanity check: Bilateral contacts

### ISSUE THREAD

**Issue by andreadelprete** [andreadelprete/consim#9]
To verify that the exponential integrator is working fine we can test it on a linear system with bilateral contacts. In this case the result of the integration should be independent of the integration time step.

I added an input parameter in the method `addContactPoint` to specify whether the contact should be unilateral. If that flag is set to false, then the contact is treated as bilateral. This means that, once that point collides with an object (whichever), the contact will never break. Moreover, all checks related to friction cones are disabled for bilateral contacts.

Using this new feature I then modified the `point_mass_simulation.py` script to compare exponential and Euler integrators in the case of a point mass system. To avoid errors due to the differences in collision detection, I started the simulation with the point mass already colliding with the ground (just an epsilon below the floor). The "outer" time step was set to 10 ms, but then I specified a different number of "inner" time steps, either 1, or 10, or 100 (which are the numbers in the figure legend). Here you can see the resulting trajectories (z axis).

![ball_height](https://user-images.githubusercontent.com/1797690/77235138-a59bf180-6bb3-11ea-86d3-df488f665f60.png)

As expected, the exponential integrators always give the same result, regardless of the number of inner time steps (1, 10, 100). "Euler 100" (i.e. with an inner time step of 0.1 ms) gave results quite similar to the exponential integrators. "Euler 10" instead gave a very different result. I didn't include "Euler 1" because it went unstable.

In conclusion, this test seems to validate the exponential integrators, and it confirms that they give exact results when applied to linear dynamical systems.
